server/routes/BCST2-form/BCST2FormController.ts

```diff
@@ -9,88 +9,100 @@ import { SubmittedInput, SubmittedQuestionAndAnswer } from '../../data/model/BCS
 export default class BCST2FormController {
   constructor(private readonly rpService: RpService) {}
 
-  getNextPage: RequestHandler = async (req, res, next): Promise<void> => {
+  getNextPage: RequestHandler = (req, res, next) => {
     const { prisonerData } = req
     const { token } = req.user
     const { pathway } = req.query
 
-    const nextPage = await this.rpService.fetchNextPage(
```

Comment:
I think you should keep the async await syntax
catch errors in the rpService.fetchNextPage and extract any potential failure point from the argument list for example `prisonerData.personalDetails.prisonerNumber as string` could be extracted as `const prisonerNumber = prisonerData?.personalDetails?.prisonerNumber?.toString() || null`